### DISCUSSION TITLE
I can't use vscode commands (like ctrl + shift + arrow keys or ctrl + X) while using the extension

### DISCUSSION THREAD

**Discussion by GoodBoyNeon** [VSCodeVim/Vim#7711] [Q&A]
I am a long time vs code user, but recently started using vim. I liked that you can do everything from the home row keyboard. So I used vim in vscide using the extension. But now, I cannot use commands like ctrl + x or strl + shift + arrow keys. Is there a configuration setting that prevents me from doing so?

**Comment by J-Fields** [comment]
See the `vim.handleKeys` section of [this table](https://github.com/VSCodeVim/Vim#vscodevim-settings) (apologies, docs are lackluster at the moment).
